circle.yml

```diff
@@ -11,6 +11,8 @@ dependencies:
     - atom -v
     - apm -v
     - apm install
-# test:
+test:
+  pre:
```

Comment:
I would rather this was changed to:
```yaml
test:
  override:
    - npm run lint
    - npm test
```

So that we are explicitly stating what we want tested instead of relying on Circle's inference behavior, but currently at least it makes no difference so it could go either way.